OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -213,6 +217,24 @@ private void fetchCourseComponent() {
         getLoaderManager().initLoader(0, null, this);
     }
 
+    private void trackAATestCourseOutline() {
+        if (!isVideoMode &&
+                environment.getConfig().getFirebaseConfig().isEnabled()) {
+            FirebaseRemoteConfig firebaseRemoteConfig = FirebaseRemoteConfig.getInstance();
+            firebaseRemoteConfig.fetchAndActivate().addOnCompleteListener(task -> {
+                String group = firebaseRemoteConfig.getString(Analytics.Keys.AA_EXPERIMENT);
```

Comment:
We should put a Success check here 
like https://github.com/firebase/quickstart-android/blob/master/config/app/src/main/java/com/google/samples/quickstart/config/java/MainActivity.java#L105

Reply:
> We should put a Success check here
> like https://github.com/firebase/quickstart-android/blob/master/config/app/src/main/java/com/google/samples/quickstart/config/java/MainActivity.java#L105

this Success condition only checks that requested data is fetched or not from the remote server.

Reply:
@farhan 
Purpose of `firebaseRemoteConfig.fetchAndActivate()` 
Ref: https://firebase.google.com/docs/reference/android/com/google/firebase/remoteconfig/FirebaseRemoteConfig.html#fetchAndActivate()